Library/Formula/glibc.rb

```diff
@@ -19,6 +21,8 @@ def install
         "--prefix=#{prefix}",
         "--enable-obsolete-rpc",
         "--without-selinux"] # Fix error: selinux/selinux.h: No such file or directory
+      args << "--enable-kernel=" +
+        `uname -r | cut -d. -f1,2,3 | cut -d- -f1` if build.with? "current-kernel"
```

Comment:
No idea. My understanding was that it still might have something at the end (i.e. `3.7.5.something`) but I may be wrong.